src/ledger/LedgerManagerImpl.cpp

```diff
@@ -1822,6 +1971,8 @@ LedgerManagerImpl::transferLedgerEntriesToBucketList(
     ltx.getAllEntries(initEntries, liveEntries, deadEntries);
     if (blEnabled)
     {
+        addAnyContractsToModuleCache(lh.ledgerVersion, initEntries);
+        addAnyContractsToModuleCache(lh.ledgerVersion, liveEntries);
```

Comment:
Is this expected to be a no-op now, or do we have the restored entries in `liveEntries`?

Reply:
I don't know -- I figured it'd be fairly future-proof to just assume both live and init _might_ carry contracts. @SirTyson do you know off hand?

Reply:
Yeah, I think it shouldn't matter much, I just found this a bit suspicious, given that contract code must be immutable. But this shouldn't hurt either.